Maple2.Model/Metadata/SetItemOptionMetadata.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Maple2.Model.Enum;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Maple2.Model.Metadata;
+
+public record SetItemInfoMetadata(
```

Comment:
as mentioned, let's ingest `setitemname.xml` as well and make this `ISearchable`